pkg/devfile/parser/data/common/command_helper.go

```diff
@@ -4,10 +4,8 @@ import "strings"
 
 // GetID returns the ID of the command
 func (dc DevfileCommand) GetID() string {
```

Comment:
Wondering if we even need `GetID` anymore? It was mostly needed when the ID wasn't stored at the top level part of the command and we had to check either `Composite` or `Exec` to get it.

But TBH, we can remove in a later PR. It's fine to leave in now.

Reply:
Yeah, let's remove that in a later PR

Reply:
I think we can keep it in, its always neat to have helper funcs even if we can get it directly now :)